lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -78,19 +78,26 @@ module.exports = function(
     }
 
     Bluebird.try(() => {
-      if (!SupportedFormatsModel.looksLikeWindowsImage(image.path)) {
-        return false;
+      let message = null;
+
+      if (SupportedFormatsModel.looksLikeWindowsImage(image)) {
+        message = messages.warning.looksLikeWindowsImage();
+      } else if (SupportedFormatsModel.missingPartitionTable(image)) {
+        message = messages.warning.missingPartitionTable();
+      }
+
+      if (message) {
+        // TODO: `Continue` should be on a red background (dangerous action) instead of `Change`.
+        // We want `X` to act as `Continue`, that's why `Continue` is the `rejectionLabel`
+        return WarningModalService.display({
+          confirmationLabel: 'Change',
+          rejectionLabel: 'Continue',
+          description: message
+        });
       }
 
-      AnalyticsService.logEvent('Possibly Windows image', image);
```

Comment:
Yeah, we shouldn't delete this, plus we should add another event to track images without partitions.